README.md

```diff
@@ -18,46 +30,242 @@ Staff interface for Palvelutarjotin
   - [`yarn test:debug`](#yarn-testdebug)
   - [`yarn build`](#yarn-build)
   - [`yarn codegen`](#yarn-codegen)
-- [Versioning](#versioning)
-- [Setting up development environment locally with docker](#setting-up-development-environment-locally-with-docker)
-  - [Set tunnistamo hostname](#set-tunnistamo-hostname)
-  - [Create a new OAuth app on GitHub](#create-a-new-oauth-app-on-github)
-  - [Install local tunnistamo](#install-local-tunnistamo)
-  - [Install Palvelutarjotin GraphQl server locally](#install-palvelutarjotin-graphql-server-locally)
-  - [palvelutarjotin-admin-ui](#palvelutarjotin-admin-ui)
-- [Debugging](#debugging)
-  - [Debugging project in VS Code](#debugging-project-in-vs-code)
-  - [Debugging Tests in VS Code](#debugging-tests-in-vs-code)
-  - [Debugging Tests in Chrome](#debugging-tests-in-chrome)
-- [Docker](#docker)
+- [Headless CMS](#headless-cms)
+  - [Headless CMS React Components -lib](#headless-cms-react-components--lib)
+- [Releases, changelogs and deployments](#releases-changelogs-and-deployments)
+  - [Conventional Commits](#conventional-commits)
+  - [Releasable units](#releasable-units)
+  - [Configuration](#configuration)
+  - [Troubleshoting release-please](#troubleshoting-release-please)
+    - [Fix merge conflicts by running release-please -action manually](#fix-merge-conflicts-by-running-release-please--action-manually)
+  - [Deployments](#deployments)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 
-## Environments
+## About Kultus
+
+**What is it?**
+A service for educators to find and book culture and leisure sectors activities from ecosystem for their group, helps providers process significantly.
+
+Make booking process easier to all participant (front and back end)
+- help educators to find activities
+- help enrolment process
+- help providers (culture and leisure department and partly metropolitan-area ecosystem) to show activities
+- gather data (equal possibilities and use to all group, use of Kuva´s work, resource needs)
+
+**Roles**
+
+- Educator can be school or kindergarden teacher who seeks educational activities for study groups.
+- Provider is third party user or user from KUVA services who wants to provide activities for educators
+- Admin manages rights and pages, modifies front page, categories and newsletter, statistics
+
```

Comment:
It would be IMHO better to somehow make this section specifically talk about what this repository is about, not what Kultus is about in general as a whole. Or at least add the specific part to this and have a the general level stuff in addition to the specific part. Now this section does not really talk about what this repository is about.

Reply:
Looking now at the README as it starts with `Providers' UI` which ties to the `Provider` term in `Who use it` section of `About Kultus` there's an apparent connection to what this repository is about. Because of this I don't think the "About Kultus" section needs to be changed anymore, this connection makes this repository's function clear.